plugins/metrics/mana.go

```diff
@@ -202,6 +201,4 @@ func measureMana() {
 		consensusAvg = consensusSum / float64(len(neighborConsensusMap))
 	}
 	averageNeighborsConsensus.Store(consensusAvg)
-
-	delegationAmount.Store(manarefresher.TotalDelegatedFunds())
```

Comment:
This is the only place where we set value of this variable. We should remove it completely from the metrics.